prow/pjutil/pjutil.go

```diff
@@ -278,3 +279,11 @@ func JobURL(plank config.Plank, pj prowapi.ProwJob, log *logrus.Entry) string {
 	}
 	return ""
 }
+
+// ClusterToCtx converts the prow job's cluster to a cluster context
+func ClusterToCtx(cluster string) string {
```

Comment:
Not 100% convinced I see the value in exposing this two line method.

Reply:
Should we keep it local in each controller?

Reply:
Don't feel super strongly about it one way or another.

Reply:
Alright, then let's reuse it like this.

Reply:
Give it a `/hold cancel` when you're ready for it to merge!

Reply:
I validated it today, and found a small issue. The local context needs to be set before truncating the config map. 

I will send this fix in a separate PR also for build controller. 

This PR is ready for merging.

Reply:
https://github.com/kubernetes/test-infra/pull/12320